ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json

```diff
@@ -9,11 +9,12 @@
   {"LiveFrom": "01/01/2017", "ID": "solsNotApplyingReasonFixedList", "ListElementCode": "PowerReserved", "ListElement": "They're holding power reserved", "DisplayOrder": 2},
   {"LiveFrom": "02/01/2017", "ID": "WillAvailability", "ListElementCode": "WillNotAvailable", "ListElement": "Will not available"},
   {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT205", "ListElement": "IHT205 - there was no inheritance tax to pay", "DisplayOrder": 1},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT207", "ListElement": "IHT207 - the person who died lived abroad and there was no inheritance tax to pay", "DisplayOrder": 2},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT400421", "ListElement": "IHT400 and IHT421", "DisplayOrder": 3},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "NA", "ListElement": "Not applicable - DO NOT USE", "DisplayOrder": 4},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormEstate", "ListElementCode": "IHT207", "ListElement": "IHT207", "DisplayOrder": 1},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormEstate", "ListElementCode": "IHT400421", "ListElement": "IHT400 and IHT421", "DisplayOrder": 2},
+  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT400", "ListElement": "IHT400 - only select this option if you applied after ((date))", "DisplayOrder": 2},
```

Comment:
think was agreed that date here would be a variable, maybe like CCD_DEF_CASE_SERVICE_BASE_URL

Reply:
too much effort